### DISCUSSION TITLE
any way to publish information out of a select many?

### DISCUSSION THREAD

**Discussion by abeWustl** [bonsai-rx/bonsai#1510] [Q&A]
I need to publish values out a few digital outputs that are nested inside a select many and subscribe to them somewhere else in the workflow. The problem is that I've been trying publish, multicast out of the select many, but I can't subscribe to them. My goal is to publish timestamps and the boolean, then subscribe to them, and then zip them to a CSV writer. Any thoughts on how to solve this problem? Thanks!

Attached is a simple version of the workflow

![image](https://github.com/bonsai-rx/bonsai/assets/91749443/2db8b698-7afe-451b-96c6-3a27b51d71ad)

[GithubSelectManyHelp.zip](https://github.com/bonsai-rx/bonsai/files/12264751/GithubSelectManyHelp.zip)

**Comment by bruno-f-cruz** [comment]
Use BehaviorSubject sources and multicast. [See an example here](https://bonsai-rx.org/docs/tutorials/state-machines.html)
